keras_retinanet/utils/eval.py

```diff
@@ -21,9 +21,45 @@
 
 import numpy as np
 import os
-
+import io
+import tensorflow as tf
+from PIL import Image
 import cv2
 
+def TensorboardImage(writer, image, number=0, step=0):
+    '''
+        writer : buffer to write the image
+        images : image OPENCV format.  we cast in uint8 because need for tensorboard buffer format.
+        
+    '''
+    
+
+    height, width, c = image.shape
+    
+        
+    # temp = images[i,:,:,:]*255
+    temp = np.copy(image)
+    #if channel == 1:
+        #temp = np.expand_dims(temp, axis=-1)
+        
+    temp = temp.astype('uint8')
+    
+        
+    if c == 3:
```

Comment:
else?

Reply:
I do not need else, because for now only channels = 3 is possible, but in the future, if multi channels is added, we will need to add an else part.